GTG/gtk/browser/tag_editor.py

```diff
@@ -243,7 +243,7 @@ def __build_window(self):
         self.tp_grid.attach(self.tn_entry, 1, 0, 1, 1)
         self.tn_cb_lbl_align = Gtk.Alignment.new(0, 0.5, 0, 0)
         self.tp_grid.attach(self.tn_cb_lbl_align, 0, 1, 1, 1)
-        self.tn_cb_lbl = Gtk.Label(label=_("Show Tag in Work View :"))
+        self.tn_cb_lbl = Gtk.Label(label=_('Show Tag in "Actionable" view:'))
```

Comment:
Unicode typography comment here as well - with the same caveat that maybe it's good it's not done yet.